tests/conftest.py

```diff
@@ -78,6 +78,12 @@ def not_a_tty():
         yield mck
 
 
+@pytest.fixture(autouse=True)
+def locale_environ(monkeypatch):
+    # ensures git output is english
+    monkeypatch.setenv('LC_ALL', 'C')
```

Comment:
I prefer
```python
with mock.patch.dict(os.environ, {'LANG': 'C.UTF-8'}):
    yield
```

as the scope is more well-defined than `monkeypatch` and it is more nice to python and its unicode support (`LANG=C` will cause default-ascii in several places)

I'm curious in the actual failures that this fixes though and this patch should be split into another PR -- could you include the failures in the issue you created?

Reply:
sure, bascially the problem is that git output was german on mysystem, i'll follow up on it the next time i touch my work pc

Reply:
oh actually! even better this project uses `pytest-env` so that can be set there!

Reply:
moved to #110